website/docs/reference/node-selection/syntax.md

```diff
@@ -103,9 +103,21 @@ As your selection logic gets more complex, and becomes unwieldly to type out as
 consider using a [yaml selector](/reference/node-selection/yaml-selectors). You can use a predefined definition with the `--selector` flag.
 Note that when you're using `--selector`, most other flags (namely `--select` and `--exclude`) will be ignored.
 
+### Troubleshoot with the `ls` command
+
+Constructing and debugging your selection syntax can be challenging.  To get a "preview" of what will be selected, we recommend using the [`list` command](/reference/commands/list).  This command, when combined with your selection syntax, will output a list of the nodes that meet that selection criteria.  The `dbt ls` command supports all types of selection syntax arguments, for example:
+
+```bash
+dbt ls --select "path.to.my.models" # Lists all models in a specific directory.
+dbt ls --select "source_status:fresher+" # Shows sources updated since the last dbt source freshness run.
+dbt ls --select state:modified+ # Displays nodes modified in comparison to a previous state.
+dbt ls --select "result:<status>+ state:modified+ --defer --state ./<dbt-artifact-path>" # Lists nodes that match certain result statuses and are modified, which can help with troubleshooting.
```

Comment:
This is a core team question on if this is valid, because I lifted this from below as well..
Either way, if we removed it, no big deal as it was just a sample example of something complicated

Reply:
Maybe that is a placeholder for SUCCESS or FAILED?

Reply:
Yep:
result:error	✅	✅	✅	✅
result:success	✅	✅	✅	
result:skipped	✅		✅	✅
result:fail				✅
result:warn				✅
result:pass

Reply:
So I think it's probably ok Maybe we need to tweak the table itself too..

Reply:
Added a commit to do that